sklearn/utils/sparsetools/setup.py

```diff
@@ -1,22 +1,15 @@
-import numpy
+# Remove in version 0.21
```

Comment:
Do we need this line in `utils/setup.py`?
https://github.com/scikit-learn/scikit-learn/pull/8854#discussion_r119776180
`config.add_subpackage('sparsetools')`

Apparently the lack of a `setup.py` and `tests` folder causes [this test](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/tests/test_common.py#L140) to fail..